code/modules/mob/living/simple_animal/vore/rous.dm

```diff
@@ -41,6 +41,121 @@
+				food = null
+			else
+				WanderTowards(S.loc)
+			break
+
+	if(!food) return
+
+	var/food_dist = get_dist(src,food)
+
+	if(food_dist > 7) //Lose interest on this person.
+		food = null
+		if(hunger < 25) //Get miffed but don't let people evade stack you into pouncing the next victim on sight.
+			hunger += 5
+		else if(hunger > 25) //Leave room for interaction with the next victim if blueballed by evader.
+			hunger = 25
+
+	if(food_dist > 1)
+		if(stance == STANCE_IDLE)
+			if(set_follow(food,10 SECONDS))
+				handle_stance(STANCE_FOLLOW)
+				if(resting)
+					lay_down()
+
+	if(food_dist <= 1)
+		if(hunger < 15)
+			if(prob(25))
+				visible_emote(pick("sniffs curiously at [food].",
+								   "stares at [food], seeming to want something.",
+								   "sniffs at [food]'s hands.",
+								   "sniffs curiously at [food]'s pockets.",
+								   "sits down for a moment, reaching towards [food] with its paws."))
+				hunger += 5
+		else if(hunger < 30)
+			if(prob(25))
+				visible_emote(pick("sniffs intently against [food], especially their pockets and gear.",
+								   "stands up to beg [food] for snacks.",
+								   "attempts to burrow into [food]'s pockets.",
+								   "leans against [food], licking its chops.",
+								   "hungrily nibbles onto [food]."))
+				hunger += 5
+		else if(hunger < 45)
+			if(prob(25))
+				visible_emote(pick("growls at [food], sounding rather hangry!",
+								   "aggressively bumps and nudges against [food], trying to make something fall out.",
+								   "salivates at [food] in an unsettling manner.",
+								   "pushes hard against [food], licking its chops.",
+								   "almost sinks its teeth into [food], just stopping to give them another chance."))
+				hunger += 5
+		else if(hunger < 50)
+			visible_emote("appears to have had enough and prepares to strike!")
+		else
+			food.Weaken(5)
+			food.visible_message("<span class='danger'>\the [src] pounces on \the [food]!</span>!")
+			target_mob = food
+			EatTarget()
+			hunger = 0
+			food = null
+
+/mob/living/simple_animal/hostile/bigrat/passive/attackby(var/obj/item/O, var/mob/user) // Feed the rat your food to satisfy it.
+	if(istype(O, /obj/item/weapon/reagent_containers/food/snacks))
+		qdel(O)
+		playsound(src.loc,'sound/items/eatfood.ogg', rand(10,50), 1)
+		hunger = 0
+		food = null
+		return
+	. = ..()
+
+/mob/living/simple_animal/hostile/bigrat/passive/Found(var/atom/found_atom)
+	if(!SA_attackable(found_atom))
+		return null
+	else if(ishuman(found_atom) && will_eat(found_atom))
+		var/mob/living/carbon/human/H = found_atom
+		for(var/obj/item/weapon/reagent_containers/food/snacks/S in H)
+			if(!food)
+				visible_emote("sniffs around the air intently, seeming to have caught a whiff of food!")
+			if(resting)
+				lay_down()
+			food = H
+			return found_atom
+			break
+	return null
+
+/mob/living/simple_animal/hostile/bigrat/passive/FindTarget()
+	var/atom/T = null
+	for(var/atom/A in ListTargets(view_range))
+		if(A == src)
```

Comment:
What's the purpose of overriding this?

Reply:
To make use of the "hostile" mob type looking for targets while idling. The parent proc would force the mob to start attacking the target as well.

Reply:
You can set the specific_targets var instead, which does basically exactly what you want. If set, it will only test Found() and ignore the rest of FindTarget.

Reply:
Yeah, this does only test for the found targets, but the parent FindTarget() will still force STANCE_ATTACK on the found target.

Reply:
Alright testing it now